src/clustering.jl

```diff
@@ -34,21 +34,21 @@ function cluster_loads!(grid::Grid, nload)
     end
     sum_pops = Symmetric(sum_pops, :L)
     sub_dist = Matrix(load_sub_dist(substations(grid), load_dist, id2ind, sum_pops))
-    const POPLIMIT = 200000 # population limit for a substation (=> 400MW load)
+    POPLIMIT = 200000 # population limit for a substation (=> 400MW load)
 
     while length(substations(grid)) > nload
         n = length(substations(grid))
         ok = false
         i = j = -1
         while true
-            m = indmin(sub_dist)
+            m = findmin(sub_dist)[2]  # [2] is the index of the smallest item
```

Comment:
`indmin` was replaced with `argmin`:
```suggestion
            m = argmin(sub_dist)
```